packages/eslint-plugin-lit-a11y/lib/rules/aria-role.js

```diff
@@ -76,20 +79,15 @@ module.exports = {
 
     return {
       TaggedTemplateExpression: node => {
-        if (
-          node.type === 'TaggedTemplateExpression' &&
-          node.tag.type === 'Identifier' &&
-          node.tag.name === 'html'
-        ) {
+        if (isHtmlTaggedTemplate(node)) {
           const analyzer = TemplateAnalyzer.create(node);
 
           analyzer.traverse({
             enterElement: element => {
-              // eslint-disable-next-line
-              for (const attr in element.attribs) {
+              for (const [attr, value] of Object.entries(element.attribs)) {
```

Comment:
did we need to traverse the entire prototype chain in these cases?

Reply:
hmm not sure, can you think of a better way to iterate in this case?

Reply:
usually when you want to iterate over an object's properties, `.keys` or `.entries` is where it's at.

typical use of `for..in` requires `if(Object.hasOwnProperty(key))` to prevent executing on prototypical properties.

so in most cases, my change is preferred. case for allowing `for ... in` with `// eslint-disable` is when you specifically want to iterate over every single property up the entire prototype chain

Reply:
ah yes yes, good change 👍